packages/devtools/src/schema/parser.ts

```diff
@@ -450,6 +451,25 @@ export class SchemaParser {
             relation: { source: 'queryCount', model, property },
           }
         }
+        case 'relationSetFrom': {
+          if (!isObjectType(type)) {
+            throw new Error(
+              `Unsupported @relationSetFrom directive on field ${fieldName} of object ${objectName}, @relationSetFrom can only be set on a referenced object`,
+            )
+          }
+          const property = directive.args?.property as string | void
+          if (property == null) {
+            throw new Error(
+              `Missing property argument for @relationSetFrom directive on field ${fieldName} of object ${objectName}`,
+            )
+          }
+          return {
+            type: 'view',
+            required: false,
```

Comment:
I thought we were going to set these fields as required

Reply:
The view is a sort of "inverse" relation, basically here we lookup if there's a document with a model using the SET account relation that points to the current document, so it shouldn't be a required field because the relation may exist or not independently of the current document, if that makes sense?